pkg/auth/handler/login.go

```diff
@@ -132,9 +113,13 @@ func (h LoginHandler) Handle(req interface{}) (resp interface{}, err error) {
 		}
 	}()
 
-	if valid := h.PasswordAuthProvider.IsLoginIDValid(payload.RawLoginID); !valid {
-		err = skyerr.NewInvalidArgument("invalid login_id, check your LOGIN_IDS_KEY_WHITELIST setting", []string{"login_id"})
-		return
+	if payload.LoginIDKey != "" {
+		loginIDMap := make(map[string]string)
+		loginIDMap[payload.LoginIDKey] = payload.LoginID
+		if valid := h.PasswordAuthProvider.IsLoginIDValid(loginIDMap); !valid {
```

Comment:
I think the validation does not need the login id value. Can we change `func IsLoginIDValid(map[string]string) bool` to `func IsLoginIDKeyValid(string) bool`?

Reply:
`IsLoginIDValid` is also used in signing up, which may have validation on login ID in future. I suggest keeping it at the moment.

Reply:
Fair enough.